explainer.md

```diff
@@ -19,44 +19,40 @@ The following code uses the ```::highlight``` pseudo-element to apply a yellow b
+The next example achieves the same result by using the style property on the HighlightsRangeGroup object. The style property allows for programmatic manipulation of the style separate from the declarative model of the ::highlight pseudo-element.
 
 ```html
 <body><span>One </span><span>two </span><span>three...</span>
 <script>
-    let highlightRange = new Range();
-    highlightRange.setStart(document.body, 0);
-    highlightRange.setEnd(document.body, 2);
-    highlightRange.style.backgroundColor = "yellow";
-    highlightRange.style.color = "blue";
+    let highlightsRange = new Range();
+    highlightsRange.setStart(document.body, 0);
+    highlightsRange.setEnd(document.body, 2);
+
+    let highlightsRangeGroup = new HighlightsRangeGroup(highlightsRange);
+    highlightsRangeGroup.style.backgroundColor = "yellow";
+    highlightsRangeGroup.style.color = "blue";
 
-    CSS.highlights.append("inline-highlight", highlightRange);
+    CSS.highlights.set("inline-highlight", highlightsRangeGroup);
 </script>
 ```
 
-
-CSS.highlights is a **HighlightsMap**. The **HighlightsMap** is a maplike object with a few affordances for mapping a name to an ordered sequence of Range objects, including a variadic ```set()```,  an ```append()``` method, and an ```insert()``` method. The order is important because multiple ranges can apply over the same content and order disambiguates which styles should applied as explained in the Application of CSS properties section.
-
-```javascript
-CSS.highlights.set("foo", range2, range3);
-CSS.highlights.append("foo", range4);
-CSS.highlights.insert("foo", 0, range1);
-console.log(CSS.highlights.get("foo")); // Logs [range1, range2, range3, range4]
-```
+CSS.highlights is a **HighlightsMap**. The **HighlightsMap** is a [maplike](https://heycam.github.io/webidl/#idl-maplike) object that maps a name to a HighlightsRangeGroup object. The presence of a HighlightsRangeGroup in the highlight map enables the contained Ranges to be styled � a HighlightsRangeGroup not added to the HighlightsMap will have no effect on rendering. HighlightsRangeGroup represents a set of Ranges and exposes a [setlike](https://heycam.github.io/webidl/#idl-setlike) interface. The text content of the contained Ranges will be styled according to the HighlightRangeGroup�s `style` property (which is a CSSStyleDeclaration). HighlightsRangeGroups can also have styles applied by the ::highlight pseudo elements, when the identifier passed to the ::highlight pseudo is the same as the name of the HighlightsRangeGroup in the HighlightsMap. The HighlightRangeGroup also has a `priority` property � this is used to disambiguate which styles should be used when text content is covered by Ranges from different HighlightsRangeGroups.
```

Comment:
In general, now that I'm reading this again, let's take the opportunity to get rid of language around 'styling Ranges' but instead 'styling text contained by Ranges', etc.

So change
"highlight map enables the contained Ranges to be styled"
to
"highlight map enables the text within the contained Ranges to be styled"

Reply:
Makes sense. I'm changing this on line 55 too "Any Range that exists in the highlight map under that identifier _will get its text styled_ based on these computed maps of the originating elements that are covered by the Range"